ocaml/idl/datamodel_pool.ml

```diff
@@ -804,6 +813,7 @@ open Datamodel_types
          ; field ~in_product_since:rel_inverness ~qualifier:DynamicRO ~ty:Bool ~default_value:(Some (VBool false)) "igmp_snooping_enabled" "true if IGMP snooping is enabled in the pool, false otherwise."
          ; field ~in_product_since:rel_quebec ~qualifier:RW ~ty:String ~default_value:(Some (VString "")) "uefi_certificates" "The UEFI certificates allowing Secure Boot"
          ; field ~in_product_since:rel_stockholm_psr ~qualifier:RW ~ty:Bool ~default_value:(Some (VBool false)) "is_psr_pending" "True if either a PSR is running or we are waiting for a PSR to be re-run"
-         ; field ~qualifier:DynamicRO ~lifecycle:[Published, rel_next, ""] ~ty:(Set (Ref _certificate)) "certificates" "CA certificates installed in the pool";
+         ; field ~qualifier:DynamicRO ~lifecycle:[Published, rel_next, ""] ~ty:(Set (Ref _certificate)) "certificates" "CA certificates installed in the pool"
+         ; field ~qualifier:RW ~in_product_since:rel_next ~lifecycle:[Published, rel_next, ""] ~ty:Bool ~default_value:(Some (VBool false)) "tls_verification_enabled" "True iff TLS certificate verification is enabled"
```

Comment:
Long term this should be enabled by default and it must not be allowed to turn it off during normal operation.

Reply:
Indeed. I believe it must always be RW unfortunately, in order to implement `Pool.emergency_disable_tls_verification`

Reply:
I think this should be `DynamicRO`. We don't want clients to set this directly, and rather use the API call above, don't we?

Reply:
> xapi itself can always write to any field

I misunderstood the meaning of DynamicRO. In this case I definitely agree with you and Pau!